chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+
+        uri_display = prelude + filename_k;
+      }
+    }
+    return uri_display;
+  }
+
+  /**
+   * Make anchored backlink
+   */
+  anchorMarkdownify(text, href, comment) {
+    let text1 = (text || '').trim();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'text1' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)